test/models/book_test.rb

```diff
@@ -0,0 +1,12 @@
+require 'test_helper'
+
+class BookTest < ActiveSupport::TestCase
+  test 'isbn10 and isbn13 should be a string of numbers' do
```

Comment:
It'd be good to add a test for the 'happy path' as well, i.e. when a user succeeds in adding a book.